source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.cc

```diff
@@ -346,7 +346,7 @@ absl::Status JsonTranscoderConfig::createTranscoder(
   method_info =
       path_matcher_->Lookup(method, path, args, &variable_bindings, &request_info.body_field_path);
   if (!method_info) {
-    return {StatusCode::kNotFound, "Could not resolve " + path + " to a method."};
+    return {StatusCode::kNotFound, "Unknown method " + method + " in the path " + path + "."};
```

Comment:
Is this not found only for wrong method? Could it be unknow "path"?

Reply:
If we want to diff unknown method and unknown path, we will need some change in the path matcher, not just modify the error message. The null method_info will not contain enough info. WDYT?

Reply:
Yes. I think that's the path we want to pursue.